oars_gb_pkg/nodes/sensing/airmar.py

```diff
@@ -41,10 +41,27 @@ def __init__(self, port="/dev/ttyUSB0", use_ros=True):
             self.true_speed_pub = rospy.Publisher('/boat/speed/true', Float32, queue_size=5)
             # Track "made good" (relative to ground and true north)
             self.track_pub = rospy.Publisher('/boat/track', Float32, queue_size=10)
+
+            ############
+
+            #changing the name of the node that is being published to
+
+            #original:
+
+            # Relative wind speed (knots) and direction
+            #self.rel_wind_pub = rospy.Publisher('/weather/wind/rel', Pose2D, queue_size=5)
+            # True wind speed and direction
+            #self.true_wind_pub = rospy.Publisher('/weather/wind/true', Pose2D, queue_size=5)
+
+            #new:
```

Comment:
This comment block can be removed. The old code is in Git, so we don't need to leave it here.